### ISSUE TITLE
Any chance of build downloads for Windows / OSX? 

### ISSUE THREAD

**Issue by e247** [r3b/phantomjs#11]
Looking to get our team using Behat and part of our product is based on Flash, this looks perfect as I've already started using PhantomJS with Behat. 

Is there any chance of getting build downloads for Windows? OSX would be nice too. 
Many thanks 
Adam

**Comment by r3b** [comment]
Yes. If you check my blog, I've posted links for a win32 and an OSX binary fairly recently. http://www.ryanbridges.org

—
ryan

**Comment by adamski** [comment]
Excellent, thank you!

**Comment by imheresamir** [comment]
Flash doesn't load for me on windows 8 x64. Tested the provided win32 binary, the phantomjs 1.4.1 win32 binary, and after compilation (of this fork) from source. Linux binary works without a hitch.

Any clues about getting flash working under windows?

**Comment by r3b** [comment]
Have you tried installing the Adobe 32-bit flash plugin?

**Comment by NabiKAZ** [comment]
@r3b your site was down: https://www.host-tracker.com/InstantCheck/3/a9eae02b-5e5d-e811-8595-0003ff73517a
